### ISSUE TITLE
Support

### ISSUE THREAD

**Issue by marocdu28** [renzuzu/renzu_housing#5]
Add and support original appartment from gta online just on these appartment props like bed etc are deleted

**Comment by marocdu28** [comment]
Like A shell but in gta online interior

**Comment by renzuzu** [comment]
good idea too, this will be a headache for sure since a lot of features.

i am having some progress in housing, 

i will update the Planned in readme once its i gather all infos.

**Comment by KingMascotte** [comment]
Actually there's a simple solution to using instances with the native IPL interiors. As well as loading into the interior via instance, make use of routing buckets. This erradicates a known issue with VOIP scripts where players can hear each other while in different instances. Heres a simple example of how you could acheieve this with routing buckets:

`local InUseRoutingBuckets = 0 
local newbucket = InUseRoutingBuckets + 1
local sourceroutebucket = GetPlayerRoutingBucket(source) -- Default is 0, so this will return 0
SetPlayerRoutingBucket(source, newbucket)

**Comment by Dokman** [comment]
offices will be available with his respective garages?

**Comment by renzuzu** [comment]
> Actually there's a simple solution to using instances with the native IPL interiors. As well as loading into the interior via instance, make use of routing buckets. This erradicates a known issue with VOIP scripts where players can hear each other while in different instances. Heres a simple example of how you could acheieve this with routing buckets:
> 
> > local InUseRoutingBuckets = 0
> > local InUseRoutingBuckets = InUseRoutingBuckets + 1
> > local sourceroutebucket = GetPlayerRoutingBucket(source) -- Default is 0, so this will return 0
> > SetPlayerRoutingBucket(source, InUseRoutingBuckets)

the housing have a routing buckets to support interiors.
and about the voip problem i already sorted out it

**Comment by mascotte-scripts** [comment]
Great!
